manifests/s/SoftMaker/FreePDF/2019/2.10/SoftMaker.FreePDF.2019.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SoftMaker.FreePDF.2019
+PackageVersion: 2.10
+PackageLocale: en-US
+Publisher: SoftMaker Software GmbH
+PackageName: FreePDF 2019

```

```suggestion
PackageName: FreePDF
```
Must match Control Panel / Apps & Features.